### PR TITLE
Run pull request cleaner on Travis

### PR DESCRIPTION
**Pull request by chrismytton**
This adds a new Rake task that gets run on Travis which goes through the open pull requests and closes any that are outdated. It uses [close_old_pull_requests](https://github.com/everypolitician/close_old_pull_requests) to do the actual work, and it replaces [pull_request_cleaner](https://github.com/everypolitician/pull_request_cleaner).

### CODE REVIEW

**Comment by tmtmtmtm** [comment]
@chrismytton I find the output from this a little confusing. When I ran it it reported:

```
Pull request 16560 is outdated. (Newest pull request is 16748)
Pull request 16592 is outdated. (Newest pull request is 16738)
Pull request 16678 is outdated. (Newest pull request is 16721)
```

This makes it look like I then need to do something else to deal with these. It would be better if this could report on what happened (i.e. that it closed them, or wasn't able to to, etc)

I'm OK with this being tidied up after it goes live, due to the time pressure on this, but a little more seriously, be it also doesn't actually seem to work (at least in some cases)

https://github.com/everypolitician/everypolitician-data/pull/16678 got a 

```
This Pull Request has been superseded by #16721 but there are non-bot commits.

@everypoliticianbot is this pull request still needed?
```
but there doesn't actually appear to _be_ any non-bot commits there.

I'm also curious about what's going to happen in cases where there actually _are_ non-bot commits? e.g. is https://github.com/everypolitician/everypolitician-data/pull/16592 going to get hundreds of comments every day asking if it should still exist?

File: .travis.yml [hunk 1]
```diff
@@ -1,3 +1,8 @@
 language: ruby
+script: bundle exec rake test close_old_pull_requests
```
Comment (tmtmtmtm) [inline .travis.yml]: this makes it look like we're adding `rake test` as well as `rake close_old_pull_requests`. I'm assuming that there was just an implied `script: bundle exec rake test` previously because we didn't supply any line at all? If so it might be useful to make that version explicit first as a separate commit, so that it's more obvious that _this_ commit is only adding the `close_old_pull_requests` part.

**Comment by chrismytton** [comment]
> @chrismytton I find the output from this a little confusing. When I ran it it reported:
> 
> Pull request 16560 is outdated. (Newest pull request is 16748)
> Pull request 16592 is outdated. (Newest pull request is 16738)
> Pull request 16678 is outdated. (Newest pull request is 16721)
> This makes it look like I then need to do something else to deal with these. It would be better if this could report on what happened (i.e. that it closed them, or wasn't able to to, etc)

Yep good point, ticketed as https://github.com/everypolitician/close_old_pull_requests/issues/7.



> I'm OK with this being tidied up after it goes live, due to the time pressure on this, but a little more seriously, be it also doesn't actually seem to work (at least in some cases)
> 
> #16678 got a
> 
> This Pull Request has been superseded by #16721 but there are non-bot commits.
> 
> @everypoliticianbot is this pull request still needed?
> but there doesn't actually appear to be any non-bot commits there.

Ah, that's a bug where I've assumed that the GitHub access token will always belong to the bot. Ticketed as https://github.com/everypolitician/close_old_pull_requests/issues/6.



> I'm also curious about what's going to happen in cases where there actually are non-bot commits? e.g. is #16592 going to get hundreds of comments every day asking if it should still exist?

Hmm, hundreds of comments a day is not ideal is it! Ticketed as https://github.com/everypolitician/close_old_pull_requests/issues/5.

**Comment by chrismytton** [comment]
> Perhaps worth adding a pointer in the commit message to how the GITHUB_ACCESS_TOKEN environment variable actually got turned into the line that's actually in the file? (Presumably just by adding a URL that describes that process)

Ah yep, will do.

**Comment by chrismytton** [comment]
Merging this now. I've fixed all the close_old_pull_request_issues except https://github.com/everypolitician/close_old_pull_requests/issues/7, which we can improve once it's live. Then I'll turn off the cleaner on Heroku so this version can run over the weekend.
